requirements.txt

```diff
@@ -37,11 +42,11 @@ flask-redis==0.4.0
 Flask-SQLAlchemy @ git+https://github.com/mitsuhiko/flask-sqlalchemy.git@500e732dd1b975a56ab06a46bd1a20a21e682262
 flupy==1.2.0
 future==0.18.3
-greenlet==2.0.1
+greenlet==2.0.2
 gunicorn==20.0.4
```

Comment:
I read in one of those issues for the gunicoen error that we saw that gunicorn should be =20.1.0 if we use eventlet==0.30.2? Should we set it to that?

Reply:
Negative.  20.1.0 is the latest stables version of Gunicorn.  I tried that version with eventlet v0.30.2 and v0.33.3 (latest), and both combinations fail.